src/styles/clipper.less

```diff
@@ -119,7 +119,6 @@
 	top: @ClipperUITopRightOffset;
 	right: @ClipperUITopRightOffset;
 	width: 100px;
-	background: @OneNotePurple;
```

Comment:
this remove in particular was because it was being overridden by the line directly below it :P

since the majority of the other removes only affect the `successPanel` (which has been restyled by design), I assume your other concern is around the removal of
`.height-animator {
    overflow: hidden;		
}`
I can try putting it back in to see if it still breaks the ratings panel render #Pending